imports.go

```diff
@@ -0,0 +1,7 @@
+//go:build tinygo.wasm
+
+package httpwasm
+
+//go:wasm-module httpwasm
```

Comment:
ps this has the WebAssembly import module as "httpwasm", but maybe it is nicer as "http_wasm"? wdyt? In any case glad it isn't "env"!

Reply:
http is far better choice thanks @anuraaga!